### PR TITLE
[Improvement][script] daemon script usage

### PR DESCRIPTION
**Pull request by huzekang**
Improvement daemon script usage .

### CODE REVIEW

**Comment by huzekang** [comment]
It is convenient to use dolphinscheduler-daemon.sh on the server

File: script/dolphinscheduler-daemon.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-usage="Usage: dolphinscheduler-daemon.sh (start|stop|status) <command> "
+usage="Usage: dolphinscheduler-daemon.sh (start|stop) <api-server|master-server|worker-server|alert-server> "
```
Comment (zhuangchong) [inline script/dolphinscheduler-daemon.sh:19]: Why remove status?

Comment (zhuangchong) [inline script/dolphinscheduler-daemon.sh:19]: There are other commands, please confirm.

**Review by dailidong — APPROVED** [review]
+1  good job

File: script/dolphinscheduler-daemon.sh (hunk 1 above)
Comment (dailidong) [reply script/dolphinscheduler-daemon.sh]: > There are other commands, please confirm.

I just append `status`

**Comment by dailidong** [comment]
thanks for your first contribution, if you need any help, please add my wechat(510570367) or mail(dailidong66@gmail.com), when added, please tell me your github id, thx

**Pull request merged**